### ISSUE TITLE
[a11y] do not read out loud initial of Avatar

### ISSUE THREAD

**Issue by bmarty** [element-hq/element-x-android#1864]
One little thing I have also found is that it would be nice to silence initial letter avatar and +number on read receipts.

For the avatar I did it like this...

```
diff --git a/libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/Avatar.kt b/libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/Avatar.kt
index 4ce4aa8f1..efa19933a 100644
--- a/libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/Avatar.kt
+++ b/libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/Avatar.kt
@@ -27,6 +27,9 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.semantics.clearAndSetSemantics
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
@@ -96,7 +99,9 @@ private fun InitialsAvatar(
         val ratio = fontSize.value / originalFont.fontSize.value
         val lineHeight = originalFont.lineHeight * ratio
         Text(
-            modifier = Modifier.align(Alignment.Center),
+            modifier = Modifier
+                .clearAndSetSemantics {}
+                .align(Alignment.Center),
             text = avatarData.initial,
             style = originalFont.copy(fontSize = fontSize, lineHeight = lineHeight, letterSpacing = 0.sp),
             color = avatarColors.foreground,
```

_Originally posted by @pvagner in https://github.com/vector-im/element-x-android/issues/1845#issuecomment-1822957514_
            

Note: `+number on read receipts` should be correctly handled by #1845 (to check)
